test/hooks-integration.test.js

```diff
@@ -63,22 +84,10 @@ describe('nion hooks: integration tests', () => {
 
     describe('nion decorator', () => {
         it('injects props into the component', async () => {
-            function Container() {
-                const returned = useNion({
-                    dataKey: 'test',
-                    endpoint: buildUrl('/test'),
-                })
-                return <div returned={returned} />
-            }
-
-            let Wrapper
-
-            act(() => {
-                Wrapper = mount(Wrap(Container))
-            })
+            const c = createTestNionComponent()
```

Comment:
could we use a more lucid variable name here and throughout? If 'Wrapper' was not appropriate,
- component
- nionComponent
- testNionComponent
....all seem more explanatory

Reply:
@dk-dk-dk-dk You're commenting on changes that aren't a part of the final PR. In any case, no, calling `getNionProps(createTestNionComponent())` each time would not result in same test behavior as `getNionProps(nionComponent)` with same value.

It has since been decided to leave **hooks-integration.test.js** alone and focus on delivering perf improvements that pass existing test suite (caveat: necessary change to denormalization tests to account for modified cache structure + removing an unnecessary flaky test)